test/integration/testdata/pilot.yaml.tmpl

```diff
@@ -81,6 +81,12 @@ spec:
         args:
         - discovery
         - --v={{.Verbosity}}
+        - -a
+        - {{.Namespace}}
+        - --registries
+        - {{.Registry}}
+        - --eurekaserverURL
```

Comment:
is this safe to supply if eureka mirror is not deployed?

Reply:
yes, the value isn't used